api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/ObservationValueTranslatorImplTest.java

```diff
@@ -35,37 +33,73 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
 import org.openmrs.Concept;
-import org.openmrs.ConceptDatatype;
 import org.openmrs.ConceptNumeric;
 import org.openmrs.Obs;
 import org.openmrs.api.ConceptService;
-import org.openmrs.api.context.Context;
-import org.openmrs.module.fhir2.api.translators.ConceptTranslator;
-import org.powermock.core.classloader.annotations.PrepareOnlyThisForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.openmrs.module.fhir2.api.dao.FhirConceptSourceDao;
+import org.openmrs.module.fhir2.api.dao.impl.FhirConceptDaoImpl;
+import org.openmrs.module.fhir2.api.dao.impl.FhirConceptSourceDaoImpl;
+import org.openmrs.module.fhir2.api.impl.FhirConceptServiceImpl;
+import org.openmrs.module.fhir2.api.impl.FhirConceptSourceServiceImpl;
 
-@RunWith(PowerMockRunner.class)
-@PrepareOnlyThisForTest({ Context.class })
+@RunWith(MockitoJUnitRunner.class)
 public class ObservationValueTranslatorImplTest {
 	
 	private static final String CONCEPT_VALUE_UUID = "12345-abcde-54321";
 	
 	private static final String OBS_STRING = "An ingenious observation";
 	
-	@Mock
-	private ConceptTranslator conceptTranslator;
+	private ConceptTranslatorImpl conceptTranslator;
 	
 	@Mock
 	private ObservationQuantityCodingTranslatorImpl quantityCodingTranslator;
 	
+	@Mock
+	ConceptService conceptService;
+	
 	private Obs obs;
 	
 	private ObservationValueTranslatorImpl obsValueTranslator;
 	
 	@Before
 	public void setup() {
-		obsValueTranslator = new ObservationValueTranslatorImpl();
+		FhirConceptDaoImpl fhirConceptDao = new FhirConceptDaoImpl();
+		fhirConceptDao.setConceptService(conceptService);
+		FhirConceptServiceImpl fhirConceptService = new FhirConceptServiceImpl();
+		fhirConceptService.setDao(fhirConceptDao);
+		FhirConceptSourceDao fhirConceptSourceDao = new FhirConceptSourceDaoImpl();
+		FhirConceptSourceServiceImpl fhirConceptSourceService = new FhirConceptSourceServiceImpl();
+		fhirConceptSourceService.setDao(fhirConceptSourceDao);
+		conceptTranslator = new ConceptTranslatorImpl();
+		conceptTranslator.setConceptService(fhirConceptService);
+		conceptTranslator.setConceptSourceService(fhirConceptSourceService);
+		obsValueTranslator = new ObservationValueTranslatorImpl() {
```

Comment:
This allows avoiding having to do a static mock on the Context, which generally leads to bad things.